PURL-TYPES.rst

```diff
@@ -577,6 +589,20 @@ swift
 
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
+	  
+vip
+---
+``vip`` for VI Package Manager (VIPM) format packages (LabVIEW):
+
+- The ``namespace`` is the company name from the package attributes (optional).
```

Comment:
It doesn't seem like the namespace is used by the native vipm tools (the website makes links like `vipm://oglib_numeric`), so it probably should not be included here.